Jellyfin.Server/Migrations/MigrationRunner.cs

```diff
@@ -13,10 +14,10 @@ public sealed class MigrationRunner
         /// <summary>
         /// The list of known migrations, in order of applicability.
         /// </summary>
-        internal static readonly IMigrationRoutine[] Migrations =
+        private static readonly Type[] _migrationTypes =
```

Comment:
What about discovering all classes that implement ``IMigrationRoutine` using reflection?

Reply:
I have it tracked as a separate task in this aggregate issue: https://github.com/jellyfin/jellyfin/issues/2538

I just implemented this small piece as a part of something else I was working on